CaptureSDK/GiniCaptureSDK/Tests/GiniCaptureSDKTests/DecimalExtensionTest.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  File.swift
+//  
+//
+//  Created by David Vizaknai on 04.04.2023.
+//
+
+import XCTest
+
+class DecimalExtensionTests: XCTestCase {
+    func testStringValueFromTwoDecimals() {
+        let decimalValue: Decimal = 8.28
+        let decimalString = decimalValue.stringValue(withDecimalPoint: 2)
+        XCTAssertEqual(decimalString, "8.28", "Decimal string value is not equal to expected string value.")
+    }
+
+    func testStringValueFromWholeNumber() {
+        let decimalValue: Decimal = 5
+        let decimalString = decimalValue.stringValue(withDecimalPoint: 2)
+        XCTAssertEqual(decimalString, "5.00", "Decimal string value is not equal to expected string value.")
+    }
+
+    func testStringValueFromOneDecimal() {
+        let decimalValue: Decimal = 210.3
+        let decimalString = decimalValue.stringValue(withDecimalPoint: 2)
+        XCTAssertEqual(decimalString, "210.30", "Decimal string value is not equal to expected string value.")
```

Comment:
```suggestion
        XCTAssertEqual(decimalString, "210.30", "Decimal string value is equal to expected string value.")
```

Reply:
The string represents the description of the failure according to the Apple documentation. So it should be correct as it is.